### PR TITLE
Java homework Lecture3/Task1 has been done

### PR DESCRIPTION
**Pull request by bumax1989**

### CODE REVIEW

**Review by anastasiaasadcheva94 — DISMISSED** [review]
lgfm

File: homework/java/BuchynskiM_JHL1T2/L3/Task1_DogDemo.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        public String getName() {
+            return name;
+        }
+
+        public void setName(String name) {
+            this.name = name;
+        }
+
+        public void voice(){
+            System.out.println(name+" is barking");
```
Comment (Basist92) [inline homework/java/BuchynskiM_JHL1T2/L3/Task1_DogDemo.java:51]: please input spaces around concotenations for more readable code

File: homework/java/BuchynskiM_JHL1T2/L3/Task1_DogDemo.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        }
+
+        public void eat(){
+            System.out.println("My name is " + name + ". I'm " +age+ " year old and i'm eating.");
```
Comment (Basist92) [inline homework/java/BuchynskiM_JHL1T2/L3/Task1_DogDemo.java:55]: same as above

File: homework/java/BuchynskiM_JHL1T2/L3/Task1_DogDemo.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        }
+
+        public void sleep() {
+            System.out.println("My color is "+ color +" and I'm going to sleep");
```
Comment (Basist92) [inline homework/java/BuchynskiM_JHL1T2/L3/Task1_DogDemo.java:59]: same as above

**Review by Basist92 — COMMENTED** [review]
some remarks for more readable code

File: homework/java/BuchynskiM_JHL1T2/L3/Task1_DogDemo.java (hunk 1 above)
Comment (bumax1989) [reply homework/java/BuchynskiM_JHL1T2/L3/Task1_DogDemo.java]: ok. Thank you. The changes has been done

File: homework/java/BuchynskiM_JHL1T2/L3/Task1_DogDemo.java (hunk 2 above)
Comment (bumax1989) [reply homework/java/BuchynskiM_JHL1T2/L3/Task1_DogDemo.java]: ok

File: homework/java/BuchynskiM_JHL1T2/L3/Task1_DogDemo.java (hunk 3 above)
Comment (bumax1989) [reply homework/java/BuchynskiM_JHL1T2/L3/Task1_DogDemo.java:59]: ok

**Review by Basist92 — APPROVED** [review]
Before and after every plus should be space for more readable code.  look on string 51 and 59 in your code. I've approved

**Comment by Basist92** [comment]
Lgfm

**Review by Basist92 — APPROVED** [review]

**Review by anastasiaasadcheva94 — APPROVED** [review]

**Review by AleksandrBohan — APPROVED** [review]
lgfm

**Pull request merged**